### PR TITLE
Add RscriptRequirement

### PR DESCRIPTION
**Pull request by Asnelchristian**
Closes https://github.com/coala-analyzer/coala/issues/2571

### CODE REVIEW

File: coalib/bears/requirements/RscriptRequirement.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+                'dependencies=TRUE)"'.format(self.flag,
+                                             self.package, self.repos))
+
+    def is_installed(self):
+        """
+        Checks if the dependency is installed.
+
+        :param return: True if dependency is installed, false otherwise.
+        """
+        return True if run_shell_command(
```
Comment (Adrianzatreanu) [inline coalib/bears/requirements/RscriptRequirement.py]: i like this!

File: coalib/bears/requirements/RscriptRequirement.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        :param package: A string with the name of the package to be installed.
+        :param version: A version string. Leave empty to specify latest version.
+        :param flag:    A string that specifies any additional flags, that
+                        are passed to the manager.
+        :param repos:   The repository from which the package to be installed is
+                        from.
+        """
+        PackageRequirement.__init__(self, 'R', package, version)
+        self.flag = flag
+        self.repos = repos
```
Comment (Adrianzatreanu) [inline coalib/bears/requirements/RscriptRequirement.py]: should it be repo? can it have more repositories?

File: coalib/bears/requirements/RscriptRequirement.py (hunk 1 above)
Comment (Adrianzatreanu) [inline coalib/bears/requirements/RscriptRequirement.py]: but apparently the circleCI doesnt like it. i think you need to install the package and R as well in ``.misc/deps.sh`` probably

File: coalib/bears/requirements/RscriptRequirement.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+        >>> RscriptRequirement(
+        ...     'formatR', '' , '-e',
+        ...     'http://cran.rstudio.com').install_command()
+        'R -e "install.packages('formatR', repos = 'http://cran.rstudio.com')"'
+
+        :param return: A string with the installation command.
+        """
+        return ('R {} "install.packages(\"{}\", repos=\"{}\", '
+                'dependencies=TRUE)"'.format(self.flag,
```
Comment (Adrianzatreanu) [inline coalib/bears/requirements/RscriptRequirement.py]: if this needs to be in the command, why is it not in the doctest?

Comment (Asnelchristian) [inline coalib/bears/requirements/RscriptRequirement.py]: ah i just forgot :)

File: coalib/bears/requirements/RscriptRequirement.py (hunk 2 above)
Comment (Asnelchristian) [inline coalib/bears/requirements/RscriptRequirement.py]: i don't think so

File: coalib/bears/requirements/RscriptRequirement.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    """
+
+    def __init__(self, package, version="", flag="", repo=""):
+        """
+        Constructs a new ``RscriptRequirement``, using the
+        ``PackageRequirement`` constructor.
+
+        >>> pr = RscriptRequirement(
+        ...         'formatR', version='1.4', flag='-e',
+        ...         repo= "http://cran.rstudio.com")
```
Comment (Adrianzatreanu) [inline coalib/bears/requirements/RscriptRequirement.py]: extra blank space after ``=``

**Comment by Adrianzatreanu** [comment]
ack 9506a5e

**Comment by Asnelchristian** [comment]
reack https://github.com/coala-analyzer/coala/pull/2572/commits/3af8f9f914eee91aad65150a9e17e92d528efb0b

**Comment by Asnelchristian** [comment]
@rultor merge

**Pull request merged**